habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java

```diff
@@ -264,4 +288,31 @@ protected void setAnchorSourceDirectory(File anchorSourceDirectory) {
     public File getAnchorOutputDirectory() {
         return anchorOutputDirectory;
     }
+
+    protected void setDockerfile(File dockerfile) {
+        this.dockerfile = dockerfile;
+    }
+
+    protected void performDockerfileUpdateForVirtualEnvironment() {
+        String updatedDockerfile =
+                DockerfileUtil.updateDockerfileWithContainerStageLogic(dockerfile,
+                DockerfileUtil.getContainerStageContentFrom(getBuilderStageTemplateFileName(), this.anchorSourceDirectory.getName(), this.singleHabushuDependencyBaseDir),
+                DockerfileUtil.getContainerStageContentFrom(getFinalStageTemplateFileName(), null, null));
```

Comment:
A: If we rebrand the `DockerfileUtil` class, it probably makes sense to push these two method calls into the `udpateDockerfile...` method.  As it is, I had to jump back and forth between the classes to understand exactly what was happening.